mlir/test/migraphx_models/resnet50-v1-7-tests/resnet50-v1-7_1.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: rocmlir-driver -kernel-pipeline migraphx,highlevel %s | rocmlir-gen -ph -print-results -rand none - | rocmlir-driver -kernel-pipeline full  -arch gfx1100 --verify-passes | rocmlir-opt
```

Comment:
Are we sure these shouldn't be e2e tests?

Reply:
Yeah, I think compilation tests are OK for now. If we opt to have e2e tests later, It would be straightforward to change the script to do that.